### DISCUSSION TITLE
Retraining an existing spacy model (scispacy)

### DISCUSSION THREAD

**Discussion by farrandi** [explosion/spaCy#8323] [Help: Coding & Implementations] [answered]
Hello I am wondering how I would train the NER from an existing spacy model (in my case scispacy) instead of a blank model. 
I managed to train an NER from a blank model because the spacy documentation in their website helps with that, but I am not sure what to do for training the NER of an existing model. 

Is it right that all there is to change is the config file? 
I tried by adding source to my config file:
```
[components.ner]
source = "en_core_sci_sm"
factory = "ner"
moves = null
update_with_oracle_cut_size = 100 
```

but it is giving me this error:
```ner -> source   extra fields not permitted

{'nlp': <spacy.lang.en.English object at 0x7f9a82938f70>, 'name': 'ner', 'model': {'@architectures': 'spacy.TransitionBasedParser.v2', 'state_type': 'ner', 'extra_state_tokens': False, 'hidden_width': 64, 'maxout_pieces': 2, 'use_upper': True, 'nO': None, 'tok2vec': {'@architectures': 'spacy.Tok2VecListener.v1', 'width': 96, 'upstream': '*'}}, 'moves': None, 'source': 'en_core_sci_sm', 'update_with_oracle_cut_size': 100, '@factories': 'ner'} 
```

Update: my program is now training in the CLI,
All I did was to remove everything under `[components.ner]`
so it became:
```
[components.ner]
source = "en_core_sci_sm"
```

**Comment by polm** [comment, marked as answer]
Glad you figured it out!
